app/main/views.py

```diff
@@ -1,7 +1,39 @@
 from flask import render_template
 from . import main
-
+from app import models, db
 
 @main.route('/')
 def index():
     return render_template('main/index.html')
+
+
+@main.route('/map')
+def get_map():
+    #Get information on all Incident Reports in the db, and pass to
```

Comment:
You should use the more traditional docstring format. Check out most other functions with comments and/or [read this](https://www.python.org/dev/peps/pep-0257/)